internal/provider/common/vdc/vdc.go

```diff
@@ -50,21 +51,31 @@ func Init(client *client.CloudAvenue, vdc types.String) (VDC, diag.Diagnostics)
 			return VDC{}, d
 		}
 	}
-	// Request vDC
-	_, vdcOut, err := client.GetOrgAndVDC(client.GetOrg(), vdc.ValueString())
+	// Request Org and vDC
+	orgOut, vdcOut, err := client.GetOrgAndVDC(client.GetOrg(), vdc.ValueString())
 	if err != nil {
 		d.AddError("Error retrieving VDC", err.Error())
 		return VDC{}, d
 	}
-	return VDC{Vdc: vdcOut}, nil
+	return VDC{Vdc: vdcOut, Org: orgOut}, nil
 }
 
-// GetName give you the name of the vDC
-func (v VDC) GetName() string {
+// Name give you the name of the vDC
+func (v VDC) Name() string {
 	return v.Vdc.Vdc.Name
 }
 
-// GetID give you the ID of the vDC
-func (v VDC) GetID() string {
+// ID give you the ID of the vDC
+func (v VDC) ID() string {
 	return v.Vdc.Vdc.ID
 }
```

Comment:
I find that in use it is more readable to keep the action of the function in its name

https://github.com/unknwon/go-code-convention/blob/main/en-US.md#functions-and-methods-1